aws/table_aws_backup_vault.go

```diff
@@ -20,7 +20,7 @@ func tableAwsBackupVault(_ context.Context) *plugin.Table {
 		Name:        "aws_backup_vault",
 		Description: "AWS Backup Vault",
 		Get: &plugin.GetConfig{
-			KeyColumns: plugin.AnyColumn([]string{"name"}),
+			KeyColumns: plugin.AllColumns([]string{"name", "region"}),
```

Comment:
@bigdatasourav Why does `region` need to be a required key qual? I looked in the linked issue, but it didn't explain it well.

Reply:
Instead of requiring a `region` key qual, I suggest we remove it and instead ignore the `AccessDeniedException` error in the `GetConfig`'s `IgnoreConfig`. It's a little unusual, but if you try to get a non-existent vault, you get this error anyway:
```
> select * from aws_backup_vault where name = 'test'
Error: 1 connection failed:
connection 'aws': rpc error: code = Unknown desc = AccessDeniedException:
	status code: 403, request id: f6fe7b3f-c4c2-4427-990c-7329ef25725f (SQLSTATE HV000)
```

We should note in the table doc that if you pass in the name of a vault that doesn't exist or you don't have access to (through the vault's access policy), you will get no rows.

Reply:
@cbruno10 When we do a list call (show all vaults in each region) -
```
> select name, region from aws_backup_vault
+--------------------------------+-----------+
| name                           | region    |
+--------------------------------+-----------+
| aws/efs/automatic-backup-vault | us-east-1 |
| Default                        | us-west-2 |
| test                           | us-west-2 |
| Default                        | us-east-1 |
+--------------------------------+-----------+
```
So the user has access to the vault test.

Now when the same user is doing a get call for the vault test (look for the vault test in each region)-
```
> select name, region from aws_backup_vault where name = 'test'
Error: 16 list calls returned errors:
 AccessDeniedException: 
	status code: 403, request id: 6e1666ee-0711-48cb-885e-d98e871813c9
AccessDeniedException: 
	status code: 403, request id: 9e855c8a-1bef-458b-ad74-415a30e3573a
AccessDeniedException: 
	status code: 403, request id: 0875c36d-9ef9-48c1-896b-d38484648125
AccessDeniedException: 
	status code: 403, request id: 63900097-ec92-4d0d-b619-53c4991c442d
AccessDeniedException: 
	status code: 403, request id: 5754bf3f-4cec-4208-ae42-a2413f350097
AccessDeniedException: 
	status code: 403, request id: 8081d952-fff5-429c-a1df-d765c0c4b270
AccessDeniedException: 
	status code: 403, request id: 0b37942e-e213-493a-ac0c-df1c2631f6eb
AccessDeniedException: 
	status code: 403, request id: 81f7e619-d28c-42f7-80e0-227c7d90a8e4
AccessDeniedException: 
	status code: 403, request id: fa46b880-138d-46c2-9ef7-5fb9862ec334
AccessDeniedException: 
	status code: 403, request id: 1fec5494-ddad-458b-9e6e-db3ed1adbe0d
AccessDeniedException: 
	status code: 403, request id: dd5c2fca-66e4-4bd7-9eb4-4892a041323a
AccessDeniedException: 
	status code: 403, request id: e4409e2a-076c-437d-beda-6dfd23fc906f
AccessDeniedException: 
	status code: 403, request id: 3337ccad-0994-4d0c-b880-26aba314cef1
AccessDeniedException: 
	status code: 403, request id: d132d5d4-52c5-413f-96fb-6558b6f2915f
AccessDeniedException: 
	status code: 403, request id: 13f6ed35-546d-4128-aab0-e0cb0f2471ac
AccessDeniedException: 
	status code: 403, request id: f5f44c2d-81bb-4e0e-961d-fc296572b483 (SQLSTATE HV000)
```
Here when the API is trying to look for the vault in those regions where it is unavailable, it is throwing `AccessDeniedException`. But the vault exists in a particular region, and the user has access. It is the Steampipe configuration where we are asking to look in each region.